pkg/cloudprovider/aws/amifamily/resolver.go

```diff
@@ -93,7 +95,7 @@ func (r Resolver) Resolve(ctx context.Context, provider *v1alpha1.AWS, nodeReque
 	if err != nil {
 		return nil, err
 	}
-	amiFamily := r.getAMIFamily(provider.AMIFamily, options)
+	amiFamily := GetAMIFamily(provider.AMIFamily, options)
```

Comment:
Does `GetAMIFamily` need to be exported? It's fine if so, but I couldn't tell from the diff.

Reply:
Yeah, it's used in the CloudProvider package when getting instance types, so we do have to export.